tests/queries/0_stateless/02561_null_as_default_more_formats.sh

```diff
@@ -7,15 +7,19 @@ CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
 $CLICKHOUSE_CLIENT -q "drop table if exists test"
 $CLICKHOUSE_CLIENT -q "create table test (x UInt64 default 42, y UInt64, z LowCardinality(String) default '42') engine=Memory";
+uid=$RANDOM$RANDOM
```

Comment:
Maybe use `$CLICKHOUSE_TEST_UNIQUE_NAME`?

Reply:
Of course. Didn't know that there is such variable.